worker/codalabworker/formatting.py

```diff
@@ -53,16 +53,16 @@ def parse_size(s):
     """
     try:
         if s[-1].isdigit():
-            return float(s)
+            return int(s)
```

Comment:
I'm trying to remember why the `float` was there - must have been a reason for it, since it's a thing that you kind of go out of your way to do.  Maybe it was to avoid overflow (though Python uses big decimals automatically anyway)?  Anyway, can't think of it.